packages/react-urql/src/context.ts

```diff
@@ -3,7 +3,7 @@ import { Client, createClient } from '@urql/core';
 
 // We assume some default options here; mainly not to actually be used
 // but not to error catastrophically if someone is just playing around
-const defaultClient = createClient({ url: '/graphql' });
+const defaultClient = createClient({ url: '/graphql', exchanges: [] });
```

Comment:
We may want to remove the defaults while we're at it, finally 🤔 
I just realised that it's technically possible to upgrade `@urql/core` without upgrading `urql`, which pretty much just fails on `composeExchanges(undefined)` then.

I'd propose that we allow `composeExchanges` to handle that, but also remove the defaults finally.

Reply:
this would already be handled by the `fallbackExchange` I removed the defaults from react/preact